src/Avalonia.NameGenerator/Generator/XamlXViewResolver.cs

```diff
@@ -59,6 +59,14 @@ IXamlAstNode IXamlAstVisitor.Visit(IXamlAstNode node)
             if (!isAvaloniaControl)
                 return node;
 
+            var type = clrType;
```

Comment:
I would move this piece of code before https://github.com/AvaloniaUI/Avalonia.NameGenerator/blob/c732f24f88c0cefcd4ab81194dd47271dc3d1ad2/src/Avalonia.NameGenerator/Generator/XamlXViewResolver.cs#L54-L57

and I would change the code like this:

```csharp
            var isAvaloniaControl = isWindow || clrType
                .Interfaces
                .Any(abstraction => abstraction.IsInterface &&
                                    abstraction.FullName == "Avalonia.Controls.IControl");
```

This is because Avalonia.Controls.Window is a subclass of Avalonia.Controls.IControl